cmd/snap-repair/cmd_run.go

```diff
@@ -0,0 +1,43 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/snapcore/snapd/i18n"
+)
+
+func init() {
+	short := i18n.G("Run repair-assertions that need to run")
+	long := i18n.G("")
+
+	if _, err := parser.AddCommand("run", short, long, &cmdRun{}); err != nil {
```

Comment:
I'm wondering if we should have the complexity of subcommands compared to simply a bare snap-repair. In fact, we should probably avoid go-flags and i18n altogether as well, and have just a barebones "snap-repair" tool with no flag parsing.  All user driven interaction with the feature should come via the "snap" tool anyway, so there's not much benefit in that extra complexity vs. the risks they bring.